src/coreclr/vm/arm64/asmhelpers.asm

```diff
@@ -78,6 +78,13 @@
         ret     lr
     LEAF_END
 
+;; uint64_t GetSveLengthFromOS(void);
+    LEAF_ENTRY GetSveLengthFromOS
+        ;; TODO-SVE: Uncomment once CI machines are updated to use MASM 14.4 or later
+        ;; rdvl    x0, 1
```

Comment:
Should this explicitly zero `x0` until the right MASM version can be used, so that it's state isn't undefined?

Reply:
>we should at least make it return 128 for windows/arm64

Returning hardcoded 128 for now.